authorization-service/src/functions/basic-authorizer/handler.ts

```diff
@@ -0,0 +1,27 @@
+import { middyfy } from '@libs/lambda';
+import { AuthorizationService } from 'src/services/authorization.service';
+
+const authorizerType = 'TOKEN';
+
+const basicAuthorizer = async (event) => {
+	const { authToken = '', methodArn } = event;
```

Comment:
The app is using basic authorization, so authToken will in format 'Basic token'. Shouldn't you firstly remove the 'Basic ' value from the authToken before passing it to the AuthorizationService?